examples/fluids/index.rst

```diff
@@ -272,16 +272,41 @@ of total energy, is given by
    :label: eq-advection
 
 with :math:`\bm{u}` the vector velocity field. In this particular test case, a
-blob of total energy (defined by a characteristic radius :math:`r_c`) is transported by
-a uniform circular velocity field. We have solved :math:numref:`eq-advection`
-with zero energy density :math:`E`, and no-flux for :math:`\bm{u}`.
-The :math:`3D` version of this problem can be run with::
+blob of total energy (defined by a characteristic radius :math:`r_c`) is transported by two
+different wind types.
 
-   ./navierstokes -problem advection
+- **Rotation**
 
-while the :math:`2D` version with::
+   In this case, a uniform circular velocity field transports the blob of total energy. We have
+   solved :math:numref:`eq-advection` with zero energy density :math:`E`, and no-flux for :math:`\bm{u}`.
```

Comment:
When you say "with zero energy density" here, do you mean for the BCs?

Reply:
Hmm! I believe so! Actually, I didn't write this part. It was already there. Maybe adding `on the boundaries` or `for BCs`  at the end of the sentence?